contracts/scripts/generateDeploymentsMarkdown.sh

```
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+function generate() { #deploymentDir #explorerUrl
+    deploymentDir=$1
+    explorerUrl=$2
+    # shellcheck disable=SC2068
+    for f in $(ls -1 $deploymentDir/*.json 2>/dev/null | grep -v ${IGNORED_ARTIFACTS[@]/#/-e } | sort); do
```

Shellcheck (0.10.0)

[warning] Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames.

(SC2010)